agent/util-scripts/unittests

```diff
@@ -574,15 +574,20 @@ function sort_tdslog {
 }
 
 function sort_tmlogs {
-    sort_log_file ${_testdir}/mock-run/tm/tm.logs
+    # Sort the tool-meister logs created by the tool data sink leveraging
+    # the sequence number ordering, but remove the sequence the number.
+    sort ${_testdir}/mock-run/tm/tm.logs \
+        | sed -e 's;^\([a-z\.0-9-]*\) \([0-9][0-9]*\) \(.*\)$;\1 \3;g' \
```

Comment:
This is OK for the moment, but there are a couple of things.
- I don't think you need the third subexpression:  if you match on the first two (with a trailing space) and replace them with the first one (plus a trailing space), then I think you've got it.  (The match doesn't need to be anchored at the end of the line.)
- The fact that either of the two expressions could be zero-length is a weakness, especially when paired with the third expression which is a total wildcard.  (E.g., it would match any line which began with two spaces.)

If you have time, you can use the `-E` switch to make `sed` use extended regular expressions, which will allow you to use `+` instead of `*` which tighten things up considerably.  Also, I think we can take advantage of the fact that the sequence number is always four digits:
```
        | sed -E -e 's;^([^[:space:]]+) ([0-9]{4}) ;\1 ;g'  \
```
(Replace one or more non-blank-space characters followed by a space followed by four digits followed by a space, anchored at the beginning of the line, with the first group followed by a space.)